src/tribler/core/components/ipv8/eva_protocol.py

```diff
@@ -412,14 +456,13 @@ async def on_acknowledgement(self, peer: Peer, payload: Acknowledgement):
             start_position = block_number * self.block_size
             stop_position = start_position + self.block_size
             data = transfer.data_binary[start_position:stop_position]
-            logger.debug(f'Transmit({block_number}). Peer hash: {hash(peer)}.')
+            logger.debug(f'Transmit({block_number}). Peer: {peer}.')
             self.community.eva_send_message(peer, Data(block_number, transfer.nonce, data))
             if len(data) == 0:
                 break
 
     async def on_data(self, peer, payload):
-        logger.debug(
-            f'On data({payload.block_number}). Peer hash: {hash(peer)}. Data hash: {hash(payload.data_binary)}')
+        logger.debug(f'On data({payload.block_number}). Peer: {peer}. Data hash: {hash(payload.data_binary)}')
         transfer = self.incoming.get(peer, None)
```

Comment:
**nitpick:** `None` is not necessary here; single-argument `get` is enough

Reply:
fixed